torch/__init__.py

```diff
@@ -574,6 +574,7 @@ def _assert(condition, message):
 import torch.futures
 import torch.nn
 import torch.nn.intrinsic
+import torch.nn.quantizable
```

Comment:
should we put this under torch.nn.mo.quantizable since we are planning to move everything under mo?

Reply:
I was thinking about it -- but currently `mo` is in the `fb` namespace. Should I start introducing the new namespace?